test/spark/gc-tests/test_scenarios.json

```diff
@@ -160,6 +160,7 @@
       }
     ],
     "file_deleted": false,
+    "direct_upload": true,
```

Comment:
Can you please edit the test description of those that now involve direct uploads to reflect that?

Reply:
I tried not to make big changes in the tests, so it really doesn't change the description - the text there still holds, I think.

Reply:
The text still holds, but if the test is failing there is no hint that this may be related to directly written objects